### PR TITLE
Modified TCK test to use junit and commented out test for calendars t…

### PR DESCRIPTION
**Pull request by albertomisail**
…hat are not julian or gregorian

### CODE REVIEW

File: jre_emul/android/platform/libcore/ojluni/src/test/java/time/tck/java/time/format/TCKDateTimeFormatter.java [hunk 1]
```diff
@@ -105,14 +106,18 @@
...
+import com.tngtech.java.junit.dataprovider.DataProviderRunner;
+import com.tngtech.java.junit.dataprovider.UseDataProvider;
+import org.junit.runner.RunWith;
+import org.junit.Test;
 
 /**
  * Test DateTimeFormatter.
  */
-@Test
+
```
Comment (tomball) [inline jre_emul/android/platform/libcore/ojluni/src/test/java/time/tck/java/time/format/TCKDateTimeFormatter.java:119]: Formatting nit: remove empty line between doc-comment and annotation.

File: jre_emul/android/platform/libcore/ojluni/src/test/java/time/tck/java/time/format/TCKDateTimeFormatter.java [hunk 2]
```diff
@@ -351,7 +357,8 @@ public void test_resolverFields_Set_null() throws Exception {
         };
     }
```
Comment (tomball) [inline jre_emul/android/platform/libcore/ojluni/src/test/java/time/tck/java/time/format/TCKDateTimeFormatter.java:358]: Close commenting of this test.

File: jre_emul/android/platform/libcore/ojluni/src/test/java/time/tck/java/time/format/TCKDateTimeFormatter.java [hunk 3]
```diff
@@ -351,7 +357,8 @@ public void test_resolverFields_Set_null() throws Exception {
...
 
-    @Test(dataProvider="formatWithZoneWithChronology")
+    @Test
```
Comment (tomball) [inline jre_emul/android/platform/libcore/ojluni/src/test/java/time/tck/java/time/format/TCKDateTimeFormatter.java:360]: If this test is commented out, dupe the previous comment. That way, it's obvious commenting out both tests is what you intended.

File: jre_emul/android/platform/libcore/ojluni/src/test/java/time/tck/java/time/format/TCKDateTimeFormatterBuilder.java [hunk 4]
```diff
@@ -81,19 +81,23 @@
...
+import com.tngtech.java.junit.dataprovider.DataProviderRunner;
+import com.tngtech.java.junit.dataprovider.UseDataProvider;
+import org.junit.runner.RunWith;
+import org.junit.Test;
 
 /**
  * Test DateTimeFormatterBuilder.
  */
-@Test
+
```
Comment (tomball) [inline jre_emul/android/platform/libcore/ojluni/src/test/java/time/tck/java/time/format/TCKDateTimeFormatterBuilder.java:94]: Remove empty line.

File: jre_emul/android/platform/libcore/ojluni/src/test/java/time/tck/java/time/format/TCKDateTimeFormatters.java [hunk 5]
```diff
@@ -101,22 +101,26 @@
...
+import com.tngtech.java.junit.dataprovider.DataProviderRunner;
+import com.tngtech.java.junit.dataprovider.UseDataProvider;
+import org.junit.runner.RunWith;
+import org.junit.Test;
 
 /**
  * Test DateTimeFormatter.
  */
-@Test
+
```
Comment (tomball) [inline jre_emul/android/platform/libcore/ojluni/src/test/java/time/tck/java/time/format/TCKDateTimeFormatters.java:114]: Remove empty line.

File: jre_emul/android/platform/libcore/ojluni/src/test/java/time/tck/java/time/format/TCKDateTimeFormatters.java [hunk 6]
```diff
@@ -1135,10 +1160,11 @@ public void remove() {
         };
     }
```
Comment (tomball) [inline jre_emul/android/platform/libcore/ojluni/src/test/java/time/tck/java/time/format/TCKDateTimeFormatters.java:1161]: End comment, and start a new one for next test.

File: jre_emul/android/platform/libcore/ojluni/src/test/java/time/tck/java/time/format/TCKLocalizedFieldPrinter.java [hunk 7]
```diff
@@ -59,27 +59,31 @@
...
+import org.junit.runner.RunWith;
+import org.junit.Test;
 
 import test.java.time.format.AbstractTestPrinterParser;
 
 /**
  * Test LocalizedFieldPrinterParser.
  */
-@Test
+
```
Comment (tomball) [inline jre_emul/android/platform/libcore/ojluni/src/test/java/time/tck/java/time/format/TCKLocalizedFieldPrinter.java:80]: Remove empty line.

File: jre_emul/android/platform/libcore/ojluni/src/test/java/time/tck/java/time/format/TCKOffsetPrinterParser.java [hunk 8]
```diff
@@ -70,14 +70,18 @@
...
+import com.tngtech.java.junit.dataprovider.DataProviderRunner;
+import com.tngtech.java.junit.dataprovider.UseDataProvider;
+import org.junit.runner.RunWith;
+import org.junit.Test;
 
 /**
  * Test DateTimeFormatterBuilder.appendOffset().
  */
-@Test
+
```
Comment (tomball) [inline jre_emul/android/platform/libcore/ojluni/src/test/java/time/tck/java/time/format/TCKOffsetPrinterParser.java:83]: Remove empty line.

File: jre_emul/android/platform/libcore/ojluni/src/test/java/time/tck/java/time/format/TCKPadPrinterParser.java [hunk 9]
```diff
@@ -61,49 +61,53 @@
...
+import com.tngtech.java.junit.dataprovider.DataProviderRunner;
+import com.tngtech.java.junit.dataprovider.UseDataProvider;
+import org.junit.runner.RunWith;
+import org.junit.Test;
 
 /**
  * Test padding behavior of formatter.
  */
-@Test
+
```
Comment (tomball) [inline jre_emul/android/platform/libcore/ojluni/src/test/java/time/tck/java/time/format/TCKPadPrinterParser.java:82]: Remove empty line.

File: jre_emul/android/platform/libcore/ojluni/src/test/java/time/tck/java/time/format/TCKResolverStyle.java [hunk 10]
```diff
@@ -66,17 +66,21 @@
...
+import com.tngtech.java.junit.dataprovider.DataProviderRunner;
+import com.tngtech.java.junit.dataprovider.UseDataProvider;
+import org.junit.runner.RunWith;
+import org.junit.Test;
 
 /**
  * Test.
  */
-@Test
+
```
Comment (tomball) [inline jre_emul/android/platform/libcore/ojluni/src/test/java/time/tck/java/time/format/TCKResolverStyle.java:82]: Remove empty line.

File: jre_emul/android/platform/libcore/ojluni/src/test/java/time/tck/java/time/format/TCKSignStyle.java [hunk 11]
```diff
@@ -64,17 +64,21 @@
...
+import com.tngtech.java.junit.dataprovider.DataProviderRunner;
+import com.tngtech.java.junit.dataprovider.UseDataProvider;
+import org.junit.runner.RunWith;
+import org.junit.Test;
 
 /**
  * Test.
  */
-@Test
+
```
Comment (tomball) [inline jre_emul/android/platform/libcore/ojluni/src/test/java/time/tck/java/time/format/TCKSignStyle.java:80]: Remove empty line.

File: jre_emul/android/platform/libcore/ojluni/src/test/java/time/tck/java/time/format/TCKTextStyle.java [hunk 12]
```diff
@@ -59,17 +59,17 @@
...
 import java.time.format.TextStyle;
 
-import org.testng.annotations.Test;
+import org.junit.Test;
 
 /**
  * Test DecimalStyle.
  */
-@Test
+
```
Comment (tomball) [inline jre_emul/android/platform/libcore/ojluni/src/test/java/time/tck/java/time/format/TCKTextStyle.java:72]: Remove empty line.

File: jre_emul/android/platform/libcore/ojluni/src/test/java/time/tck/java/time/format/TCKZoneIdPrinterParser.java [hunk 13]
```diff
@@ -75,14 +75,18 @@
...
+import com.tngtech.java.junit.dataprovider.DataProviderRunner;
+import com.tngtech.java.junit.dataprovider.UseDataProvider;
+import org.junit.runner.RunWith;
+import org.junit.Test;
 
 /**
  * Test DateTimeFormatterBuilder.appendZoneId().
  */
-@Test
+
```
Comment (tomball) [inline jre_emul/android/platform/libcore/ojluni/src/test/java/time/tck/java/time/format/TCKZoneIdPrinterParser.java:88]: Remove empty line.

**Comment by tomball** [comment]
Merged in https://github.com/google/j2objc/commit/dba649a98af9e72c0a5e15dbeff7457601e35817

**Comment by albertomisail** [comment]
I don't think https://github.com/google/j2objc/pull/1289/commits/1ff538044c186e53f2800ac0919423f41f4ae19a was included in PR #1289 so was not merged in dba649a

**Pull request closed without merging**